plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiConfig.java

```diff
@@ -36,7 +36,7 @@
 public class HudiConfig
 {
     private List<String> columnsToHide = ImmutableList.of();
-    private boolean metadataEnabled = false;
+    private boolean metadataEnabled; // default value of false initialized
```

Comment:
Can remove the comment. It's self-explanatory. I guess you removed the init because of static code analysis failure?

Reply:
Yes, will remove.